implementations/rust/ockam/ockam_api/CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.68.0 - 2024-05-30
+
+### Added
+
+- Create a portal for exporting traces when a project exists
+- Address review comments
```

Comment:
This line can be removed, I should have squashed the PR

Reply:
Updating

Reply:
Done